scripts/content.js

```diff
@@ -22,11 +21,12 @@ fetch(proxyurl + url) // https://cors-anywhere.herokuapp.com/https://example.com
   .then(contents => {
     resp = JSON.parse(contents);
     brands = resp;
+
     const finalResult = brands.result.map((data)=>{
-    const result =   webbrands.filter((wellspentBrand)=>{
-        return wellspentBrand===data.BrandName
+    const result =   webBrands.filter((webBrand)=>{
+        return webBrand.includes(data.BrandName.toLowerCase())
       })
-      return result;
+      return result.length > 0 ? [data.BrandName.toLowerCase()] : [];
```

Comment:
To avoid considering a substring  is a brand name we should check if the world comes at the beginning of the string if so this word is a brand name, I checked inside the filter: 
```js
  const result =   webBrands.filter((webBrand)=>{
      const splitedBrand= data.BrandName.toLowerCase().split(" ")[0];
      if(webBrand.indexOf(splitedBrand)==0){
        return webBrand.includes(data.BrandName.toLowerCase())
      }
  })
```

Reply:
I change it thanks @shahenazmonia :1st_place_medal:

Reply:
I think this is unnecessary and bad. It is very very unlikely that we will accidentally match a brand to a substring and in fact when we do it probably means it is of the correct brand - eg Polo Ralph Lauren _should_ match Ralph Lauren.

Reply:
look at this string 'ASOS SUPPLY triangle abstract printed rug' when applying the code without doing something for a substring, some of Asos brand will have colour degrees related to TED brand!
I think we should find a way to handle both things